index.html

```diff
@@ -61,19 +61,19 @@ <h1>
       </section>
       <section class="readings-container">
         <article class="readings">
-          <h2 class="readings-title">Temperature</h2>
+          <h2 class="readings-title" id="temperature">Temperature</h2>
```

Comment:
An id for the title has been given which we don't want to apply changes to but to the readings value.(the p tag)